### DISCUSSION TITLE
Lost account settings after close browser and delete cache

### DISCUSSION THREAD

**Discussion by fthoma-coding** [dani-garcia/vaultwarden#2466] [Q&A] [answered]
### Subject of the issue

Lost account settings after close browser and delete cache

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.24.0
* Web-vault version: v2.25.1
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.35.4
* Clients used: 20
* Reverse proxy and version: Traefik v2.6.6
* Other relevant information: 

### Steps to reproduce

Log in to your account, go to your account settings and open the option section. Change anything... I have noticed the bug, if I changed the option "Enable full width layout" and change the theme to dark. If I hit enter, everthing changed like expected. But after closing the browser and automated cleaning of the browser cache, every setting changed back to default at the next login.

### Expected behaviour

Every account setting should persist.

### Actual behaviour

Every account setting is set to default after closing the browser and delete the cache.

**Comment by Masgalor** [comment, marked as answer]
This is normal behaviour.
All user settings are stored locally in your browser cache there is no serverside storage for this configurations.

**Comment by BlackDex** [comment]
These settings are stored within the browsers local storage. If you clear those they are obviously gone.

Those settings are not stored server side.
